patterns/testimonials-review-large-image-right.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Review with large image on right
+ * Slug: twentytwentyfive/testimonials-review-large-image-right
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
```

Comment:
Ideally, this outer group should be set to full width, so that if we add a background color it goes to the screen's edges.

https://github.com/user-attachments/assets/a244ea6b-e798-489c-9993-a201f0548d8f

Reply:
Thanks, updated in https://github.com/WordPress/twentytwentyfive/pull/118/commits/6b65a0c983ace36d4f148cd61f8410f4b5ca5744